templates/ammos-cubs-ait.template.yaml

```diff
@@ -73,6 +95,78 @@ Mappings:
       RHELHA84HVM: ami-0cca63ccd9a87f1b2
 
 Resources:
+  EfsSecurityGroup:
+    Type: AWS::EC2::SecurityGroup
+    Properties:
+      VpcId:
+        Ref: VPCID
+      GroupDescription: "EFS Security Group"
+      SecurityGroupIngress:
+        - IpProtocol: tcp
+          FromPort: 2049
+          ToPort: 2049
+          CidrIp: 0.0.0.0/0
+
+  FileSystem:
+    Type: AWS::EFS::FileSystem
+    Properties:
+      AvailabilityZoneName: !Ref 'AvailabilityZone'    
+      Encrypted: true
+      PerformanceMode: generalPurpose
+      BackupPolicy:
+        Status: DISABLED
+      FileSystemTags:
+      - Key: Name
+        Value: ait-efs
+
+  MountTargetPublicA:
+    Type: AWS::EFS::MountTarget
+    Properties:
+      FileSystemId: !Ref FileSystem
+      SubnetId: "subnet-03a1ec6be37d72106"
+      SecurityGroups:
+      - Ref: EfsSecurityGroup
+      
+  RootAccessPoint:
+    Type: AWS::EFS::AccessPoint
+    Properties:
+      FileSystemId: !Ref FileSystem
+      PosixUser:
+        Uid: "1001"
+        Gid: "1001"
+      RootDirectory:
+        CreationInfo:
+          OwnerGid: "1001"
+          OwnerUid: "1001"
+          Permissions: "777"
+        Path: "/efs"
+
+  AitEfsBootstrap:
+    Type: AWS::CloudFormation::CustomResource
+    Properties:
+      ServiceToken: !GetAtt CopyAitDeploymentEfs.Arn
```

Comment:
will need to add a `BucketName` properties or whatever you end up using in the lambda function for identifying config bucket